ddtrace/tracer/tracer_test.go

```diff
@@ -2318,21 +2319,26 @@ func TestUserMonitoring(t *testing.T) {
 
 	// This tests data races for trace.propagatingTags reads/writes through public API.
 	// The Go data race detector should not complain when running the test with '-race'.
+	wg := &sync.WaitGroup{}
+	wg.Add(2)
```

Comment:
⚠️ This should be moved into the `t.Run` section below. Otherwise running individual tests (e.g. `go test -run 'TestUserMonitoring/nested' ./ddtrace/tracer/`) will hang indefinitely.

Reply:
OOPS yes of course. I... don't understand why I put this outside the test now. Pushing a change to do this now thanks.